### PR TITLE
Routes updated and removed comments

### PR DESCRIPTION
**Pull request by anuragganjiwale**

### CODE REVIEW

File: app/views/categories/show.html.haml [hunk 1]
```diff
@@ -10,5 +10,5 @@
         .card-body
           %h5.card-title.text-center= @categories[n].category_name
           %p.card-text.text-muted.text-center= @categories[n].category_description
-          %button.btn.btn-success.text-center{:style => "color: white; display:block;"}= link_to 'Click Here',category_animation_data_path(@categories[n].id)
+          %button.btn.btn-success.text-center{:style => "color: white; display:block;"}= link_to 'Click Here',animations_path(@primary_category.id,@categories[n].id)
```
Comment (SagareGanesh) [inline app/views/categories/show.html.haml]: use space after comma

File: config/routes.rb [hunk 2]
```diff
@@ -27,10 +27,4 @@
     end
   end
 
-  namespace :api do
```
Comment (SagareGanesh) [inline config/routes.rb]: why v2, we need v1 only

Comment (anuragganjiwale) [reply config/routes.rb]: v2 is not present

Comment (anuragganjiwale) [reply config/routes.rb]: It has been removed already

**Pull request merged**